app/models/identity.rb

```diff
@@ -0,0 +1,3 @@
+class Identity < ActiveRecord::Base
+  belongs_to :user
```

Comment:
You should probably use a validation here to check for the presence of `user_id`

Reply:
Oh thanks @seafr 
I did change it.